docs/user-guide/connectors/slack.rst

```diff
@@ -17,10 +17,16 @@ Getting Credentials
 
   1. To create the app go to: https://api.slack.com/apps and click
      on *"Create New App"*.
-  2. Activate the following features: interactive components, event
-     subscriptions, bot users, permissions (for basic functionality
-     you should subscribe to the ``message.channel``,
-     ``message.groups``, ``message.im`` and ``message.mpim`` events)
+  2. Activate the following features: 
+    1. interactive components
+    2. event subscriptions
+    3. bot users
+    4. permissions (for basic functionality you should subscribe to the 
+       following events
+       ``message.channel``,
+       ``message.groups``, 
+       ``message.im`` 
+       and ``message.mpim``)
```

Comment:
following events: ``message.channel``,``message.groups``, ``message.im`` and ``message.mpim``)